### ISSUE TITLE
Values of type short can not be plotted in the plot-interfaces.

### ISSUE THREAD

**Issue by agy-why** [django-daiquiri/daiquiri#50]
## Context

I send the following query on gaia.aip.de:

```SQL
SELECT solution_id, designation, source_id, 
matched_observations, visibility_periods_used, 
parallax, parallax_error, 
astrometric_n_obs_al, 
astrometric_n_obs_ac, 
astrometric_n_good_obs_al, astrometric_n_bad_obs_al, 

FROM gdr2.gaia_source 

WHERE random_index BETWEEN 100 AND 120 
AND duplicated_source = false 
AND astrometric_params_solved = 31 

LIMIT 10
```

In order to see a possible correlation between the quality of observation and the parallax_error.

## Problem

I wanted to plot `matched_observations` vs. `parallax_error`. As the interface properly tells me it is not possible because of the type of `matched_observations`: 
```
    Columns of the type short can not be plotted
```

It could be a nice enhancement to allow such plotting.

**Comment by agy-why** [comment]
@kimakan Here are the requierments for the plots:

3 type of plots:
1. multi-line plots: in order to compare various series to each other
2. histogram plots: to check statistics from query
3. 2D-historgram plots: to quick visualize cone search results: x:ra y:dec and color: 3rd quantity.

It should be able to plot: short, int, long, double but also char, varchar, bool and dates for histogram

Multi lines:
allow to choose a quantity for x-axis
allow to add/remove lines (max 5?) 
allow to choose a quantity, color, symbol for each line
automatically add a legend
allow zooming
allow switch between x/y-lin log (with checkbox)

Histogram plot:
allow to over plot a normal distribution as line: center, sigma, amplitude
allow to select a 3rd quantity to make a selection: two histograms, one for entries with filter=True and one with filter=False
3rd quantity can be Bool (True/False) or (Var)Char mapping value1 or value2 or Decimal mapping above/bellow a given threshold.
Automatically add a legend

2-D historgram plot:
choose quantity for x-axis
choose quantity for y-axis
Choose quantity for color scale (give choice between two color scale: linear and log)
automatically add color scale legend

Sky Plot:
Choose coordinates: ra/dec or lon/lat (galactic)
Plot object accordingly in a sky projection 
choose color scale with 3rd quantity

For all, allow OpenGL as default (everything above 1000 points is slow otherwise).

If you have questions feel free to ask.

**Comment by agy-why** [comment]
It would be nice to switch on/off the various plots with a SETTING value in daiquiri:

```python
QUERY_PLOTS = {'multilines': True,
               'histohram': True,
               '2d-histogram': True,
               'skyplot': True
}
```

**Comment by kimakan** [comment]
I close the issue since the initial problem has been solved with #100
